fdbserver/storageserver.actor.cpp

```diff
@@ -1297,11 +1297,11 @@ ACTOR Future<Void> getKeyValues( StorageServer* data, GetKeyValuesRequest req )
 
 		if( req.debugID.present() )
 			g_traceBatch.addEvent("TransactionDebug", req.debugID.get().first(), "storageserver.getKeyValues.AfterVersion");
-			//.detail("ShardBegin", printable(shard.begin)).detail("ShardEnd", printable(shard.end));
+		//.detail("ShardBegin", shard.begin).detail("ShardEnd", shard.end);
```

Comment:
Any idea what the intent of this line is? There are a few of these following `g_traceBatch.addEvent` lines, but they wouldn't work without a `TraceEvent` from somewhere. Maybe the intent is that someone would modify the `addEvent` line to work? It seems a little weird to me, I'm wondering if we should just delete them.

Reply:
I agree, this is a bit weird. But I think this is a fundamental problem: we have so much commented out code and I don't like that. There's a reason we use version control after all....

If it would be after me we would either come up with some framework that uses compiler flags to activate and deactivate these kind of things or remove all these comments in the first place...

Reply:
I think a lot of the comments are for trace events that we want to be able to conveniently add in. Maybe we should make some of those SevDebug now that doing so should be cheap and then uncomment them.

For these particular lines, I think I'll probably remove them, but we don't need to do that in this PR.